trafficcloud/pm.py

```diff
@@ -81,6 +82,15 @@ def create_project_dir(self, uuid):
             print("Project exists. No new project created.")
 
     def _write_to_project_config(self):
+        # Copy information given by the project_name.cfg generated client side
```

Comment:
the loading of a config file, and doing try/except statements, is somewhat handled in `app_config.check_project_cfg_option`.  I was having a hard time making this function work, so I just used the raw `.read`, `.get`, `try/except` calls.